tests/compose-tests/test-misc-tweaks.sh

```diff
@@ -21,9 +21,16 @@ pysetjsonmember "add-files" '[["foo.txt", "/usr/etc/foo.txt"],
                               ["baz.txt", "/usr/share/baz.txt"],
                               ["bar.txt", "/etc/bar.txt"]]'
 pysetjsonmember "postprocess-script" \"$PWD/postprocess.sh\"
+pysetjsonmember "postprocess" '["""#!/bin/bash
+touch /usr/share/postprocess-testing""",
+"""#!/bin/bash
+rm /usr/share/postprocess-testing
+touch /usr/share/postprocess-testing-done"""]'
 cat > postprocess.sh << EOF
 #!/bin/bash
 set -xeuo pipefail
+# Ordering should be after postprocess
+rm /usr/share/postprocess-testing-done
```

Comment:
Ahh OK, so these tests are verifying ordering by relying on `rm` failing if the file is already missing. Which is fine, though... now that I look, we don't actually have a test that verifies that errors in the postprocess script cause a failure at compose time. We can do that separately though!